Jenkinsfile

```diff
@@ -186,6 +186,54 @@ pipeline {
                         }
                     }
                 }
+            stage('Fault injection') {
+                agent {
+                    label 'ci_vm1'
+                }
+                options {
+                    timeout(time: 60, unit: 'MINUTES')
+                }
+                steps {
+                    provisionNodes NODELIST: env.NODELIST,
+                        node_count: 1,
+                        snapshot: true
+                    runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
+                        script: """set -x
+                            . ./.build_vars-Linux.sh
+                            CART_BASE=\${SL_PREFIX%/install*}
+                            NODELIST=$nodelist
+                            NODE=\${NODELIST%%,*}
+                            trap 'set +e; set -x; ssh -i ci_key jenkins@\$NODE "set -ex; sudo umount \$CART_BASE"' EXIT
+                            ssh -i ci_key jenkins@\$NODE "set -x
+                                set -e
+                                sudo mkdir -p \$CART_BASE
+                                sudo mount -t nfs \$HOSTNAME:\$PWD \$CART_BASE
+                                cd \$CART_BASE
+                                ln -s /usr/bin/fusermount install/Linux/bin/fusermount3
+                                pip3.4 install --user tabulate
+                                ./test/iof_test_alloc_fail.py"
+                            """
```

Comment:
This looks like it could be DRYed out in a function.

Reply:
DRYed?

I agree, I was thinking about defining functions in the IOF Jenkinsfile and being able to call them from the CaRT Jenkinsfile at some point?

Reply:
> DRYed?

*D*o *N*ot  *R*epeat yourself.

But I think you got that.

> I agree, I was thinking about defining functions in the IOF Jenkinsfile and being able to call them from the CaRT Jenkinsfile at some point?

￼I'm not sure about pulling functions across `Jenkinsfile`s, but if you need functions across pipelines, that's what [pipeline-lib](https://github.com/daos-stack/pipeline-lib) is for.